### PR TITLE
feat: create ros2-ws image that has ros2-control installed

### PR DESCRIPTION
**Pull request by domire8**
## Description

This PR also targets the refactor branch, but will be squashed. This is now the Dockerfile that builds on top of the ros2-control image by just adding labels.

## Review guidelines

Estimated Time of Review: 4 minutes

## Checklist before merging:

- [ ] Confirm that the relevant changelog(s) are up-to-date in case of any user-facing changes

### CODE REVIEW

File: ros2_ws/build.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-IMAGE_NAME=aica-technology/ros2-ws
-BASE_TAG=humble
+IMAGE_NAME=gchr.io/aica-technology/ros2-ws
```
Comment (domire8) [inline ros2_ws/build.sh:3]:
```suggestion
IMAGE_NAME=ghcr.io/aica-technology/ros2-ws
```

File: ros2_ws/Dockerfile [hunk 2]
```diff
@@ -1,108 +1,10 @@
...
-
-
-FROM environment-variables as final
-COPY --from=base-workspace / /
-
-# start as ROS user
-USER ${USER}
-WORKDIR ${ROS2_WORKSPACE}
+ARG BASE_TAG=iron
+FROM ghcr.io/aica-technology/ros2-control:${BASE_TAG}}
```
Comment (domire8) [inline ros2_ws/Dockerfile:2]:
```suggestion
FROM ghcr.io/aica-technology/ros2-control:${BASE_TAG}
```

**Comment by domire8** [comment]
image

The labels seem to be here. Also, I packaged a backend with this base image and there seem to be no problem :+1:

**Review by eeberhard — APPROVED** [review]
so just to clarify, the point is to re-tag the existing (and at this point unchanging) `ros2-control` image as `ros2-ws` with the addition of the metadata labels. Then afterwards, we can revert ros2-ws Dockerfile to fully build the base workspace and install ros2-control deps all in one. And we are doing that because rebuilding ros2-ws from scratch now pulls updated (and breaking) changes to ros2 iron itself.

**Pull request merged**